build.sbt

```diff
@@ -1,8 +1,15 @@
 name := "firrtl-diagrammer"
 
-version := "1.0"
-
-scalaVersion := "2.12.6"
+resolvers ++= Seq(
+  Resolver.sonatypeRepo("snapshots"),
+  Resolver.sonatypeRepo("releases")
+)
+organization := "edu.berkeley.cs"
+version := "1.0-SNAPSHOT"
+autoAPIMappings := true
+scalaVersion := "2.11.6"
```

Comment:
Could we make the "default" `scalaVersion` 2.12.7 (i.e., same as FIRRTL), and the `crossScalaVersions` ` Seq("2.12.7", "2.11.12")`?